site/features/reactor.html

```diff
@@ -27,23 +27,35 @@ <h1>FusionAuth Reactor</h1>
     <div class="cards row">
       <div class="col-xs-12 col-lg-4 feature-card">
         <img src="/assets/img/icons/breached-password-detection.svg" alt="Breached Password Detection">
-        <h2>Breached Password Detection</h2><span class="orange stamp">NEW!</span>
+        <h2>Breached Password Detection</h2>
         <p>
-          FusionAuth will check user login credentials against a set of leaked databases that we've collected from all over the Internet (including the dark web). You can enable Breached Password Detection during account creation, password resets, password changes and login. You can then force the user to change their password or notify them via email. FusionAuth will also mark the user's account as using a breached password until it is changed. This can be used for reporting and analytics.
+          FusionAuth will check user login credentials against a set of leaked databases that we've collected from all over the Internet (including the dark web). You can enable Breached Password Detection during account creation, password resets, password changes and login. You can then force the user to change their password or notify them of the breach via email. 
```

Comment:
Dark web has almost become a cliche..., what if we said something like :

```
we've collected from all of the Internet, even the really hard to reach places.
```


I'm fine with dark web as well, if everyone else likes it.